doc/source/release-notes/Percona-Server-5.6.40-84.0.rst

```diff
@@ -0,0 +1,156 @@
+.. rn:: 5.6.40-84.0
+
+============================
+|Percona Server| 5.6.40-84.0
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.40-84.0 on
+May 30, 2018 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.40-84.0/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.40
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-40.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.40-84.0 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.40-84.0
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.40-84.0>`_.
+
+New Features
+============
+
+* A new string variable :variable:`version_suffix` allows to change suffix
```

Comment:
Can we describe some context for why we added this?  When might a user need this?

Reply:
The original use case was 3rd party software checking the version string and only continuing if they find "-enterprise" there.